microscope/filterwheels/aurox.py

```diff
@@ -86,16 +97,44 @@ class Clarity(microscope.devices.FilterWheelBase):
                   __GETSERIAL: 4,
                   __FULLSTAT: 10}
 
-    def __init__(self, **kwargs):
+    def __init__(self, camera=None, **kwargs) -> None:
+        # Extract kwargs for camera device.
+        cam_kw_keys = [k for k in kwargs if k.startswith("camera.")]
+        cam_kwargs = {}
+        for key in cam_kw_keys:
+            cam_kwargs[key.replace("camera.", "")] = kwargs[key]
+            del kwargs[key]
```

Comment:
Would a `camera_kwargs` argument with the arguments to construct the camera make this cleaner?

    def __init__(self, camera=None, camera_kwargs={}, **kwargs):
        super().__init__(**kwargs)
        if camera is not None:
            self._camera = camera(**camera_kwargs)
            ...

Reply:
It might make the code a bit cleaner, but then you'd have nested ```{ {} }``` in the config, and maybe that's not cleaner / more error-prone.